images/docs-builder/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+if [ $# -lt 1 ]; then
+	echo "You need to provide an argument [build_docs, build_docs_pdf, check_links, docs_serve]"
+fi
+
+case $1 in
+	build_docs)
+		echo "Generating html from docs source"
+		shift 
+		asciidoctor $@
+		;;
+	build_docs_pdf)
+		echo "Generating pdf from docs source"
+		shift 
+		asciidoctor-pdf $@
+		;;
+	check_links)
+		echo "Checking if all links are alive in docs source"
+		find . -name \*.adoc | xargs -n1 asciidoc-link-check -q
+		;;
+	docs_serve)
+		cd build
+		echo "Make sure port 8088 of the is exposed"
```

Comment:
`of the`? Is this a log to tell what's going on? or something the user has to fix?

Reply:
Just to tell whats going on..